### ISSUE TITLE
Bug: Key Facts Block Issue

### ISSUE THREAD

**Issue by ZoeBeykirch** [hlxsites/vg-volvotrucks-us#777]
### Issue Type

Bug

### Description

The blue text in the Key Facts block is now smaller than the original design specifications. Please compare the current implementation with the Figma design to see the intended font size.

Additionally, a pop-up message appears stating that this block is not supported. This is confusing, as the Key Facts block is listed under V2 blocks in the Sidekick library and should be fully supported for the redesign style.

### Affected Device Classes

Mobile, Tablet, Desktop

### Affected Browsers/Devices and Versions

All browsers and devices. 

### Steps to Reproduce

1. Navigate to where the Key Facts block is being used on production. EX: https://www.volvotrucks.us/trucks/all-new-vnl/#powertrain
2. Compare the text size with the Figma design: https://www.figma.com/design/02CNUdjDefh83K5OeFx9PI/Volvo-Component-Library-C%26T?node-id=1675-142&t=wRJS5x0l1o9q5XNB-1
3. View how the text size for the blue numbers is much smaller in production than in the intended designs. 

4. Also navigate to the Key Facts block in the sidekick library: https://main--vg-volvotrucks-us--hlxsites.aem.page/tools/sidekick/library.html?plugin=blocks&path=/block-library/blocks/v2-key-facts&index=1
5. View how the title is V2 Key Facts block, indicating that it should be supported for the redesign style
6. Navigate to: https://main--vg-volvotrucks-us--hlxsites.aem.page/trucks/all-new-vnl/#powertrain
7. When content editors preview the Key Facts block on a V2 page in production, it triggers the pop-up message for unsupported blocks (seen in the below screen shot). 

### Expected Results

1. The text size should match the specifications in Figma: https://www.figma.com/design/02CNUdjDefh83K5OeFx9PI/Volvo-Component-Library-C%26T?node-id=1675-142&t=wRJS5x0l1o9q5XNB-1

![Image](https://github.com/user-attachments/assets/23256040-81dd-4d07-9b4c-fe8f8f0a8144)

2. The pop-up message indicating the block is unsupported should not appear when content editors preview the page. 

### Actual Results

![Image](https://github.com/user-attachments/assets/e3e6a8e2-d055-4cbf-80f9-ac3aedf1607a)

![Image](https://github.com/user-attachments/assets/63ba3ad6-c52a-4bb0-83c4-1fac3829e3cb)

**Comment by BeckyMedlin** [comment]
@Rapostu @mollyaljenkins We need to inject this item into the sprint as it is impacting production. I'd like @amglebihan to work on the fix. He can put the Sidekick work on pause until this is handled. Thank you!

**Comment by amglebihan** [comment]
@BeckyMedlin I've looked into the issue, and the bug was caused by having a page for a v2-key-facts block in the sidekick library, even though we don't have any v2-key-facts block in our codebase. In the library, the v2-key-facts block is actually a key-facts block (v1) with the v2-redesign style applied in the metadata. This isn't sufficient for proper styling. The solution would be to create a dedicated v2-key-facts block and style it accordingly.
